modules/monitoring-exploring-the-visualized-metrics.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/examining-cluster-metrics.adoc
+
+[id="exploring-the-visualized-metrics_{context}"]
+= Exploring the visualized metrics
+
+After running the queries, the metrics are displayed on the interactive plot. The X axis of the plot represents time. The Y axis represents the metrics values. Each metric is shown as a colored graph. You can manipulate the plot and explore the metrics.
+
+.Procedure
+
+. Initially, all metrics from all enabled queries are shown on the plot. You can select which metrics are shown.
+* To hide all metrics from a query, click the action button of the query and click "Hide all series".
+* To hide a specific metric, go to the query table and click the colored square near the metric name.
+. To zoom into the plot and change the shown time range, do one of the following:
++
+--
+* Visually select the time range by clicking and dragging on the plot horizontally.
+* Use the menu in the left upper corner to select the time range.
+--
++
+To reset the time range, click "Reset Zoom".
+. To see outputs of all queries at a specific point in time, hold the mouse cursor on the plot at that point. Query outputs will appear in a popup box.
+. For more detailed information about metrics of a specific query, expand the table of that query. Every metric is shown with its current value.
+. To hide the plot, use the "Hide Graph" button.
```

Comment:
s/use the "Hide Graph" button./click **Hide Graph**./

Reply:
Implemented.